js/google_ima.js

```diff
@@ -1572,6 +1641,17 @@ require("../html5-common/js/utils/utils.js");
           case eventType.USER_CLOSE:
           case eventType.SKIPPED:
           case eventType.COMPLETE:
+            if(_usingAdRules)
+            {
+              this.adResponseTime = new Date().valueOf();
+            }
+            var adSkipped = false;
+            if (adEvent.type == eventType.SKIPPED)
```

Comment:
We try to do === on everything unless it specifically needs a ==

Reply:
fixed